sdk/storage/azure-storage-blob-nio/README.md

```diff
@@ -318,7 +318,7 @@ This project has adopted the [Microsoft Open Source Code of Conduct][coc]. For m
 [storage_account_create_portal]: https://docs.microsoft.com/azure/storage/common/storage-quickstart-create-account?tabs=azure-portal
 [identity]: https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/identity/azure-identity/README.md
 [error_codes]: https://docs.microsoft.com/rest/api/storageservices/blob-service-error-codes
-[samples]: https://docs.oracle.com/javase/tutorial/essential/io/fileio.html
+[samples]:  https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/storage/azure-storage-blob-nio/src/samples/java/com/azure/storage/blob/nio
```

Comment:
Why are you changing this line? @gapra-msft can you please check?

Reply:
@amishra-dev - This link is wrong

Reply:
That link is actually correct

Reply:
I'll go ahead and close the issue. This points to the official nio docs since this library is an implementation of that interface

Reply:
OK, I will close this PR.